src/server/services/product.service.js

```diff
@@ -1,13 +1,33 @@
 import productRepo from '../repositories/product.repository';
 
-export const getOneById = async (id) => await productRepo.getOneById(id);
-export const getAll = () => productRepo.getAll();
+// функція повертає 1 продукт або пусту стрічку якщо такого продукту немає
+export const getOneById = async (id) => {
+  const product = await productRepo.getOneById(id);
+  return product;
+};
+export const getAll = async () => {
+  try {
+    const result = await productRepo.getAll();
+    return result;
+  } catch (err) {
+    return `Can't get products:\n "${err.message}"`;
```

Comment:
here we have the same issue with consistency. You return array if everything is ok and string if it is not. you can return an empty array instead, or throw a new Error here with an appropriate message.